assignments/assignment01.rb

```diff
@@ -69,10 +102,50 @@ def my_reverse(s)
 # that determines whether a string is a palindrome
 def palindrome?(s)
   # your implementation here
+  def toChars(s)
+    #puts "toChars called"
+    s = s.downcase
+    ans = ''
+    s.each_char do |c|
+      #puts "c = #{c}"
+      #puts ".include = #{'abcdefghijklmnopqrstuvwxyz'.include? c}"
+
+      if 'abcdefghijklmnopqrstuvwxyz'.include? c     #ignore non-letter characters
+        #puts "c found to be a letter"
+        ans = ans + c
+      end
+    end
+    #puts "ans = #{ans}"
+    return ans
+  end
+
+=begin
```

Comment:
Again, this is way more complicated than necessary. All you really need to do is compare the string with non-alpha chars remove to its reverse.